asv_bench/benchmarks/omnisci/benchmarks.py

```diff
@@ -13,47 +13,45 @@
 
 """General Modin on OmniSci backend benchmarks."""
 
+import modin.pandas as pd
```

Comment:
why? also in Modin before 0.9 this would initialize the engine and backend...

Reply:
Now I am using the module defined in the `IMPL` variable.
This side-effects shouldn't affect omnisci benchmarks.